lib/policyEvaluator/RequestContext.ts

```diff
@@ -256,7 +257,7 @@ export default class RequestContext {
         this._securityToken = securityToken;
         this._policyArn = policyArn;
         this._action = action;
-        this._needQuota = actionNeedQuotaCheck[apiMethod] === true
+        this._needQuota = needQuota || actionNeedQuotaCheck[apiMethod] === true
```

Comment:
(Not for this PR)
In the cleanup to come, I'm thinking we should probably make this class dumber and leave the decision up to cloudserver